tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserSBytePerfTests.cs

```diff
@@ -0,0 +1,270 @@
+                    {
+                        PrimitiveParser.InvariantUtf8.TryParseSByte(utf8ByteSpan, out sbyte value, out int bytesConsumed);
+                        TestHelper.DoNotIgnore(value, bytesConsumed);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        private static void PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength()
+        {
+            int textLength = s_SByteTextArray.Length;
+            byte[][] utf8ByteArray = (byte[][])Array.CreateInstance(typeof(byte[]), textLength);
+            for (var i = 0; i < textLength; i++)
+            {
+                utf8ByteArray[i] = Text.Encoding.UTF8.GetBytes(s_SByteTextArray[i]);
+            }
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        ReadOnlySpan<byte> utf8ByteSpan = utf8ByteArray[i % textLength];
+                        PrimitiveParser.InvariantUtf8.TryParseSByte(utf8ByteSpan, out sbyte value, out int bytesConsumed);
+                        TestHelper.DoNotIgnore(value, bytesConsumed);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("107")] // standard parse
+        [InlineData("127")] // max value
+        [InlineData("0")]
+        [InlineData("-128")] // min value
+        [InlineData("147")]
+        [InlineData("2")]
+        [InlineData("105")]
+        [InlineData("-111")]
+        [InlineData("-21")]
+        [InlineData("-2")]
+        [InlineData("-13")]
+        [InlineData("-8")]
+        [InlineData("-83")]
+        [InlineData("+127")]
+        [InlineData("+21")]
+        [InlineData("+2")]
+        [InlineData("00000000000000000000123abcdfg")]
+        [InlineData("2abcdefghijklmnop")]
+        [InlineData("14abcdefghijklmnop")]
+        [InlineData("-14abcdefghijklmnop")]
+        [InlineData("-21abcdefghijklmnop")]
+        [InlineData("-2abcdefghijklmnop")]
+        [InlineData("+14abcdefghijklmnop")]
+        [InlineData("+21abcdefghijklmnop")]
+        [InlineData("+2abcdefghijklmnop")]
+        [InlineData("+111abcdefghijklmnop")]
+        [InlineData("+000000000000000000123abcdfg")]
+        private static void PrimitiveParserByteSpanToSByte_BytesConsumed_Baseline(string text)
+        {
+            byte[] utf8ByteArray = Text.Encoding.UTF8.GetBytes(text);
+            var utf8ByteSpan = new ReadOnlySpan<byte>(utf8ByteArray);
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        int.TryParse(text, out int value);
```

Comment:
Does it make sense for the baseline to be int.TryParse when comparing to sbyte parser?